src/session/chat/history.rs

```diff
@@ -327,21 +328,41 @@ impl History {
 
     fn remove(&self, message_id: i64) {
         let self_ = imp::History::from_instance(self);
-        let index = self_.list.borrow().iter().position(|m| {
-            if let ItemType::Message(message) = m.type_() {
-                if message.id() == message_id {
-                    return true;
-                }
-            }
 
-            false
-        });
+        if let Some(message) = self_.message_map.borrow_mut().remove(&message_id) {
+            let message_ymd = glib::DateTime::from_unix_utc(message.date() as i64)
```

Comment:
This needs a conversion to local time. This should be the same as [this](https://github.com/melix99/telegrand/blob/a846f055d5414c735119850fe86f25acff80ff56/src/session/chat/item.rs#L95).

Reply:
I just tested this roughly: It doesn't seem to be necessary at all, because glib::DateTime takes into account possible different time zones when comparing two instances. Probably the comparison happens based on unix time. If I'm wrong, reopen this.

Reply:
> Probably the comparison happens based on unix time

Yeah, you're right. Actually that review was about the use of `ymd()`, but I forgot to remove it after the suggestion of the other review. 😅